Description: Drop redundant message using non-existing variable ctr.
 Python rejected the code without this patch.
Author: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-02-01
License: Apache 2 (same as caldav-tester itself)

Index: caldav-tester-packaging/src/manager.py
===================================================================
--- caldav-tester-packaging.orig/src/manager.py	2016-02-09 11:00:05.416918450 +0000
+++ caldav-tester-packaging/src/manager.py	2016-02-09 11:00:05.412918402 +0000
@@ -220,8 +220,6 @@
         if self.posttest is not None:
             self.posttest = _loadFile(self.posttest, False)
 
-        self.message("load", None, ctr + 1, len(testfiles))
-
 
     def readCommandLine(self):
         sname = "scripts/server/serverinfo.xml"
